css/templates/e-commerce/page.css

```diff
@@ -330,12 +330,15 @@ amp-selector [option][selected] {
   border-bottom: 1px solid var(--color-separator);
 }
 
-.ampstart-navbar-trigger {
-  position: absolute;
-  top: 0;
-  padding-top: var(--space-2);
+.ampstart-headerbar .ampstart-navbar-trigger {
   color: var(--color-primary);
   font-size: var(--h4);
+  /* Overrides pr2 default in navbar */
```

Comment:
Note that this is done here rather than the (navbar template)[https://github.com/raisaveuc/ampstart/blob/e20ca6709eed5a6c93f86e8d593106d5d30178c0/components/navbar/navbar.snip.html#L22] to minimize the impact on other pages that use this component